python/taichi/core/util.py

```diff
@@ -251,17 +251,17 @@ def check_exists(src):
         )
 
 
-def prepare_sandbox(src):
-    global g_tmp_dir
-    check_exists(src)
+def prepare_sandbox():
+    '''
+    return a runtime temporary directory name, which will be automatically
+    deleted on exit.
```

Comment:
```suggestion
    Returns a temporary directory, which will be automatically deleted on exit.
    It may contain the taichi_core shared object or some misc. files.
```